Assets/Shopify/UIToolkit/ShopControllers/BaseShopController.cs

```diff
@@ -42,6 +43,16 @@ public abstract class ShopControllerBase : MonoBehaviour, IShopCredentials {
         }
         [SerializeField] private string _accessToken;
 
+        /// <summary>
+        /// The Apple Merchant ID for processing Apple Pay payments.
+        /// </summary>
+        public string AppleMerchantID {
+            set { Cart.AppleMerchantID = value; } 
+            get { return Cart.AppleMerchantID; }
+        }
```

Comment:
Instead of proxying here, can we invert the dependency? (cart is initialized with the apple merchant id from this class?)

Reply:
Or if the class is serialized, you can use `FindProperty("Cart.AppleMerchantID")` in the editor to expose it without having to edit this class at all. The Cart object is serialized so therefore it will persist any changes made to it.